### DISCUSSION TITLE
Why austin doesn´t get enough recognition?

### DISCUSSION THREAD

**Discussion by PabloRuizCuevas** [P403n1x87/austin#237] [General]
I tried almost all profilers out there already and I can say that Austin is one of the better ones, thanks for that! but if you look in google for "python profiler" is hard to find his existence if you don´t know it already.

Of course this question may look silly, but Austin is not anymore a novel product and is still really hard to find, make the experiment in a private window or something.

**Comment by P403n1x87** [comment]
@PabloRuizCuevas Many thanks for your sincere appreciation of Austin 😊 . I have tried my best to share this project far and wide on many fora and platforms. I can only hope it keeps the momentum it has gained lately 🤞 https://seladb.github.io/StarTrack-js/#/preload?r=p403n1x87,austin. Word-of-mouth might contribute, so if you don't mind you could share Austin with your network and hope it resonates with others 🙂
